burnman/tools.py

```diff
@@ -643,3 +644,109 @@ def check_eos_consistency(m, P=1.e9, T=300., tol=0.01, verbose=False):
             print('Not satisfied all EoS consistency constraints for {0:s}'.format(m.to_string()))
             
     return consistency
+
+
+def _pad_ndarray_inverse_mirror(array, padding):
```

Comment:
Looking a bit more carefully, If I really wanted to add a mode I have to burrow down through scipy:
gaussian_filter -> gaussian_filter1d -> correlate1d. I append my previous statement by saying I'd really rather not do that :/